packages/suite/src/components/suite/modals/confirm/UnverifiedAddress/index.tsx

```diff
@@ -18,20 +18,23 @@ const Actions = styled.div`
 `;
 
 type Props = {
-    device: TrezorDevice;
     address: string;
     addressPath: string;
     onCancel: () => void;
 };
 
-const ConfirmUnverifiedAddress = ({ device, address, addressPath, onCancel }: Props) => {
-    const { isLocked } = useDevice();
+const ConfirmUnverifiedAddress = ({ address, addressPath, onCancel }: Props) => {
+    const { device, isLocked } = useDevice();
     const isDeviceLocked = isLocked();
     const { showAddress, showUnverifiedAddress, applySettings } = useActions({
         showAddress: receiveActions.showAddress,
         showUnverifiedAddress: receiveActions.showUnverifiedAddress,
         applySettings: deviceSettingsActions.applySettings,
     });
+
+    // just to make TS happy
+    if (!device) return null;
```

Comment:
If device is never supposed to be null, you can use the non-null assertion operator below. See: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

Not sure if that totally applies here (might be better to do it your way).

Reply:
I know, yet I would rather return null than adding bunch of ! every time when accessing `device`. There certainly is better solution, but I don't think it is non-null assertion. Current approach while weird is at least consistent across codebase 😬

Reply:
Agreed, resolving comment.